
[32] Interpretation of differential gene expression results of RNA-Seq data: review and integration.
Adam McDermaid<sup>\*</sup>, Brandon Monier<sup>\*</sup>, Jing Zhao, Bingqiang Liu, **Qin Ma**<sup>$</sup>
*Briefings in Bioinformatics*
[[Abstract]](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bby067/5066173?guestAccessKey=41fde1ae-81d4-42c5-89fd-9db5a58f8547) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-Interpretation-of-differential-gene-expression-results-of-RNA-seq-data-review-and-integration.pdf)

---

[31] A new machine learning-based framework for mapping uncertainty analysis in RNA-Seq read alignment and gene expression estimation.
Adam McDermaid, Xin Chen, Yiran Zhang, Cankun Wang, Shaopeng Gu, Juan Xie, Qin Ma<sup>$</sup>
*Frontiers in Genetics*
[[Abstract]](https://www.frontiersin.org/articles/10.3389/fgene.2018.00313/full) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-A-New-Machine-Learning-Based-Framework-for-Mapping-Uncertainty-Analysis-in-RNA-Seq-Read-Alignment-and-Gene-Expression-Estimation.pdf)


---


[30] It is time to apply biclustering: a comprehensive review of biclustering applications in biological and biomedical data.
Juan Xie, Anjun Ma, Anne Fennell, **Qin Ma**<sup>$</sup>, Jing Zhao<sup>$</sup>
*Briefings in Bioinformatics*
[[Abstract]](https://academic.oup.com/bib/article/20/4/1450/4911545?guestAccessKey=24bb8438-44be-4466-93bd-2847a1b92704) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-It-is-time-to-apply-biclustering-a-comprehensive-review-of-biclustering-applications-in-biological-and-biomedical-data.pdf)

---

[29] Bioinformatics tools for quantitative and functional metagenome and metatranscriptome data analysis in microbes.
Sheng-Yong Niu<sup>\*</sup>, Jinyu Yang<sup>\*</sup>, Adam McDermaid, Jing Zhao, Yu Kang, **Qin Ma**<sup>$</sup>
*Briefings in Bioinformatics*
[[Abstract]](https://academic.oup.com/bib/article/19/6/1415/3805128) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-Bioinformatics-tools-for-quantitative-and-functional-metagenome-and-metatranscriptome-data-analysis-in-microbes.pdf)

---


[28] DMINDA 2.0: integrated and systematic views of regulatory DNA motif identification and analyses.
Jinyu Yang, Xin Chen, Adam McDermaid, **Qin Ma**<sup>$</sup>
*Bioinformatics*
[[Abstract]](https://academic.oup.com/bioinformatics/article/33/16/2586/3611274) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-DMINDA-2.0-integrated-and-systematic-views-of-regulatory-DNA-motif-identification-and-analyses.pdf)


---

[27] QUBIC: a Bioconductor package for qualitative biclustering analysis of gene co-expression data.
Yu Zhang<sup>\*</sup>, Juan Xie<sup>\*</sup>, Jinyu Yang, Anne Fennell, **Qin Ma**<sup>$</sup>
*Bioinformatics*
[[Abstract]](https://academic.oup.com/bioinformatics/article/33/3/450/2572040) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-QUBIC-a-bioconductor-package-for-qualitative-biclustering-analysis-of-gene-co-expression-data.pdf)

---


[26] LncFinder: an integrated platform for long non-coding RNA identification utilizing sequence intrinsic composition, structural information and physicochemical property.
Siyu Han, Yanchun Liang, **Qin Ma**, Yangyi Xu, Yu Zhang, Wei Du, Cankun Wang, Ying Li<sup>$</sup>
*Briefings in Bioinformatics*
[[Abstract]](https://academic.oup.com/bib/article/20/6/2009/5062950) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-LncFinder-an-integrated-platform-for-long-non-coding-RNA-identification-utilizing-sequence-intrinsic-composition-structural-information-and-physicochemical-property.pdf)

---

[25] Physical properties of DNA may direct the binding of nucleoid-associated proteins along the E. coli genome.
Guoqing Liu<sup>$</sup>, **Qin Ma**, Ying Xu<sup>$</sup>
*Math Biosci*
[[Abstract]](https://www.sciencedirect.com/science/article/pii/S0025556417305746?via%3Dihub) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-Physical-properties-of-DNA-may-direct-the-binding-of-nucleoid-associated-proteins-along-the-E.-coli-genome.pdf)


---

[24] A Central Edge Selection Based Overlapping Community Detection Algorithm for the Detection of Overlapping Structures in Protein-Protein Interaction Networks.
Fang Zhang<sup>\*</sup>, Anjun Ma<sup>\*</sup>, Zhao Wang, **Qin Ma**, Bingqiang Liu, Lan Huang<sup>$</sup>, Yan Wang<sup>$</sup>
*Molecules*
[[Abstract]](https://www.mdpi.com/1420-3049/23/10/2633/htm) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-A-Central-Edge-Selection-Based-Overlapping-Community-Detection-Algorithm-for-the-Detection-of-Overlapping-Structures-in-Protein%E2%80%93Protein-Interaction-Networks.pdf)


---

[23] RECTA: Regulon Identification Based on Comparative Genomics and Transcriptomics analysis.
Xin Chen<sup>\*</sup>, Anjun Ma<sup>\*</sup>, Adam McDermaid, Huayuan Zhang, Cao Liu, Huansheng Cao,
**Qin Ma**<sup>$</sup>
*Genes*
[[Abstract]](https://www.mdpi.com/2073-4425/9/6/278) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-RECTA-Regulon-Identification-Based-on-Comparative-Genomics-and-Transcriptomics-Analysis.pdf)


---

[22] Evolutionary Divergence of TNL Disease-Resistant Proteins in Soybean (Glycine max) and Common Bean (Phaseolus vulgaris).
Surendra Neupane, **Qin Ma**, Febina M. Mathew, Adam J. Varenhorst, Ethan J. Andersen, Madhav P. Nepal<sup>$</sup>
*Biochemical Genetics*
[[Abstract]](https://link.springer.com/article/10.1007%2Fs10528-018-9851-z) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-Evolutionary-Divergence-of-TNL-Disease-Resistant-Proteins-in-Soybean-Glycine-max-and-Common-Bean-Phaseolus-vulgaris.pdf)


---

[21] MRHCA: a nonparametric statistics based method for hub and co-expression module identification in large gene co-expression network.
Yu Zhang<sup>$</sup>, Sha Cao, Jing Zhao, Burair Alsaihati, Qin Ma<sup>$</sup>, Chi Zhang<sup>$</sup>
*Quantitative Biology*
[[Abstract]](https://link.springer.com/article/10.1007/s40484-018-0131-z) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-MRHCA-a-nonparametric-statistics-based-method-for-hub-and-co-expression-module-identification-in-large-gene-co-expression-network.pdf)


---


[20] DOOR: A microbial operon database for gene organization and function discovery.
Huansheng Cao<sup>\*</sup>, **Qin Ma**<sup>\*</sup>, Xin Chen, Ying Xu<sup>$</sup>
*Briefings in Bioinformatics*
[[Abstract]](https://academic.oup.com/bib/article/20/4/1568/4055922?guestAccessKey=06eac434-f3e0-47d7-b916-8ba28eeec7d2) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/DOOR2017.pdf)

---


[19] An algorithmic perspective of de-novo cis-regulatory motif finding based on ChIP-seq data.
Bingqiang Liu, Jinyu Yang, Yang Li, Adam McDermaid, **Qin Ma**<sup>$</sup>
*Briefings in Bioinformatics*
[[Abstract]](https://academic.oup.com/bib/article-abstract/19/5/1069/3064185?redirectedFrom=fulltext) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/algorithm_perspective_2017.pdf)

---

[18] SeqTU: A Web Server for RNA-seq based Transcription Unit Identification in Bacteria.
Xin Chen<sup>\*</sup>, Wen-Chi Zhou<sup>\*</sup>, **Qin Ma**<sup>$</sup> and Ying Xu<sup>$</sup>
*Scientific Reports*
[[Abstract]](https://www.nature.com/articles/srep43925) [[Full paper]](https://www.nature.com/articles/srep43925.epdf?author_access_token=fcIfrxcxf657fT10X-rWXNRgN0jAjWel9jnR3ZoTv0MdvY37YSz4h0cXM-kCnj_jM9p2IXKERkVtEyZXeBLd4-gebq0bLWheudwCm2qipT4vqLRh-KreDOouu0yZrJtj)


---

[17] Systems-Level Understanding of the Impact of Ethanol-Induced Stresses and Adaptation in E. coli.
Huansheng Cao<sup>\*</sup>, Wei Du<sup>\*</sup>, Yuedong Yang, Yu Shang, Gaoyang Li, Yaoqi Zhou, Qin Ma<sup>$</sup>, Ying Xu<sup>$</sup>
*Scientific Reports*
[[Abstract]](https://www.nature.com/articles/srep44150) [[Full paper]](https://www.nature.com/articles/srep44150.pdf)


---

[16] Scalable Data Mining Algorithms in Computational Biology and Biomedicine.
Quan Zou, Dariusz Mrozek, **Qin Ma**<sup>$</sup>, Yungang Xu<sup>$</sup>
*BioMed Research International*
[[Abstract]](https://www.hindawi.com/journals/bmri/aip/5652041/) [[Full paper]](http://downloads.hindawi.com/journals/bmri/2017/5652041.pdf)


---

[15] RNA-TVcurve: A Web Server for RNA Secondary Structure Comparison based on a Multi-Scale Similarity of its Triple Vector Representation.
Ying Li, Shi Xiaohu, Liang Yanchun, Juan Xie, Yu Zhang, Qin Ma<sup>$</sup>
*BMC Bioinformatics*
[[Abstract]](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1481-7) [[Full paper]](https://www.readcube.com/articles/10.1186/s12859-017-1481-7?author_access_token=5y28DvJqS0X7Wp2GYyFYxG_BpE1tBhCbnbw3BuzI2RNieVkgREjiY85nDocVeWM6GNnr71KC-w0QK0DspJSPgyDclIpihJ4wcrZSwy3ZCTgjDTBrReKRsS1lDc99bnkwjIRZe2eQ6PA49ty0U3XSFw%3D%3D)


---

[14] GAAP: Genome-Organization-Framework-Assisted Assembly Pipeline for the Prokaryotic Genome.
Lina Yuan<sup>\*</sup>, Yang Yu<sup>\*</sup>, Yanmin Zhu, Yulai Li, Changqing Li, Rujiao Li, **Qin Ma**, Gilman Kit-Hang Siu, Jun Yu, Taijiao Jiang, Jingfa Xiao, Yu Kang<sup>$</sup>
*BMC Genomics*
[[Abstract]](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-3267-0) [[Full paper]](https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-016-3267-0)


---

[13] Bibliometric Screening of Helicobacter Pylori Pathogenic Genes Through Pathway Enrichment and Operon Analysis.
Guangyu Xu, Yan Xia, Ying Tang, Huansheng Cao, **Qin Ma**, Ying Xu, Nan Zhang, Hong Xu<sup>$</sup>
*Clinical Laboratory*
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/28164669/) [[Full paper]](https://www.clin-lab-publications.com/article/2285)


---

[12] An integrative and applicable phylogenetic footprinting framework for cis-regulatory motifs identification in prokaryotic genomes.
Bingqiang Liu, Hanyuan Zhang, Chuan Zhou, Guojun Li, Anne Fennell, Guanghui Wang, Yu Kang, Qi Liu, **Qin Ma**<sup>$</sup>
*BMC Genomics*
[[Abstract]](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2982-x) [[Full paper]](https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-016-2982-x)


---

[11] A new insight into species from the same descendents aspect and an application to Clostridia.
Zhenjia Wang, Chuan Zhou, Bingqiang Liu, **Qin Ma**, Guojun Li<sup>$</sup>
*Current Bioinformatics*
[[Abstract]](https://www.eurekaselect.com/143343) [[Full paper]](https://www.eurekaselect.com/143343)


---

[10] A link density clustering algorithm based on automatically selecting density peaks for overlapping community detection.
Guishen Wang, Lan Huang, Yan Wang, Wei Pang, **Qin Ma**<sup>$</sup>
*International Journal of Modern Physics B*
[[Abstract]](https://ui.adsabs.harvard.edu/abs/2016IJMPB..3050167H/abstract) [[Full paper]](https://www.worldscientific.com/doi/pdf/10.1142/S0217979216501678)


---

[9] Deciphering determining factors rather than microhomology on the occurrence of in-frame mutation in human CRISPR-based gene knockout.
Guohui Chuai, Fayu Yang, Yanan Chen, **Qin Ma**, Chi Zhou, Chenyu Zhu, Feng Gu, Qi Liu<sup>$</sup>
*Molecular Therapy-Nucleic Acids*
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/27301063/) [[Full paper]](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5022128/pdf/mtna201635a.pdf)


---

[8] Bacterial regulon modeling and prediction based on systematic cis regulatory motif analyses.
Bingqiang Liu, Chuan Zhou, Guojun Li, Hanyuan Zhang, Erliang Zeng, **Qin Ma**<sup>$</sup>
*Scientific Reports*
[[Abstract]](https://www.nature.com/articles/srep23030) [[Full paper]](https://www.nature.com/articles/srep23030.pdf)


---

[7] Nitrogen remobilization and conservation, and underlying senescence‐associated gene expression in the perennial switchgrass Panicum virgatum.
Jiading Yang, Eric Worley, **Qin Ma**, Jun Li, Ivone Torres-Jerez, Yi-Ching Lee, Jiyi Zhang, Nick Krom, Fuqi Liao, Yuhong Tang, Patrick X. Zhao, Michael Udvardi<sup>$</sup>
*New Phytologist*
[[Abstract]](https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.13898) [[Full paper]](https://u.osu.edu/bmbl/files/2020/08/nph.13898.pdf)


---

[6] The understanding of circular RNAs as special triggers in carcinogenesis.
Zhuoyuan Xin, **Qin Ma**, Guoqing Wang, Shuangchun Ren, Fan Li<sup>$</sup>
*Briefings in functional genomics*
[[Abstract]](https://academic.oup.com/bfg/article/16/2/80/2453159) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2016-The-understanding-of-circular-RNAs-as-special-triggers-in-carcinogenesis.pdf)


---

[5] Identifying anti-cancer drug response related genes using integrative analysis of transcriptomic and genomic variations with cell line-based drug perturbations.
Yi Sun, Yunqin Chen, Wei Zhang, **Qin Ma**, Jia Wei, Qi Liu<sup>$</sup>
*Oncotarget*
[[Abstract]](https://www.oncotarget.com/article/7012/) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2016-Identifying-anti-cancer-drug-response-related-genes-using-an-integrative-analysis-of-transcriptomic-and-genomic-variations-with-cell-line-based-drug-perturbations.pdf)


---

[4] Link community detection based on linear graphs with a novel link similarity measure.
Guishen Wang, Lan Huang, Yan Wang, **Qin Ma**, Wei Pang<sup>$</sup>
*International Journal of Modern Physics B*
[[Abstract]](https://www.worldscientific.com/doi/abs/10.1142/S0217979216500235?src=recsys&) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2016-Link-community-detection-based-on-linear-graphs-with-a-novel-link-similarity-measure.pdf)


---

[3] Revisiting Operons: An Analysis of the Landscape of Transcriptional Units in E. coli.
**Qin Ma**, Xizeng Mao, Bingqiang Liu, Zheng Chang, Chuan Zhou, Hanyuan Zhang, Ying Xu<sup>$</sup>
*BMC Bioinformatics*
[[Abstract]](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0805-8) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2015-Revisiting-operons-an-analysis-of-the-landscape-of-transcriptional-units-in-E.-coli.pdf)


---


[2] Genome-scale identification of cell-wall related genes in Arabidopsis based on co-expression network analysis.
Shan Wang<sup>\*</sup>, Yanbin Yin<sup>\*</sup>, **Qin Ma**, Xiaojia Tang, Kan Bao, Dongyun Hao, Ying Xu<sup>$</sup>
*BMC Plant Biology*
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/22877077/) [[Full paper]](https://link.springer.com/content/pdf/10.1186%2F1471-2229-12-138.pdf)

---


[1] Analysis of strand-specific RNA-seq data using machine learning reveals the structures of transcription units in Clostridium thermocellum.
Wen-Chi Chou<sup>\*</sup>, **Qin Ma**<sup>\*</sup>, Shihui Yang, Sha Cao, Dawn M. Klingeman, Steven D. Brown, Ying Xu<sup>$</sup>
*Nucleic Acids Research*
[[Abstract]](https://academic.oup.com/nar/article/43/10/e67/2409010) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2015-Analysis-of-strand-specific-RNA-seq-data-using-machine-learning-reveals-the-structures-of-transcription-units-in-Clostridium-thermocellum.pdf)
