
[28] Gene Regulatory Networks for Lignin Biosynthesis in Switchgrass (Panicum virgatum).
Xiaolan Rao, Xin Chen, Hui Shen, **Qin Ma**, Guifen Li, Yuhong Tang, Maria Pena, William York, Taylor Frazier, Scott Lenaghan, Xirong Xiao, Fang Chen and Richard A. Dixon<sup>$</sup>
*Plant Biotechnology Journal*
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/30133139/) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-Gene-regulatory-networks-for-lignin-biosynthesis-inswitchgrassPanicum-virgatum.pdf)

---

[27] SGL-SVM: a novel method for tumor classification via support vector machine with sparse group Lasso.
Yanhao Huo, Lihui Xin, Chuanze Kang, Minghui Wang, **Qin Ma**<sup>$</sup>, Bin Yu<sup>$</sup>
*Journal of Theoretical Biology*
[[Abstract]](https://www.sciencedirect.com/science/article/pii/S0022519319304679) [[Full paper]](https://u.osu.edu/bmbl/files/2020/08/1-s2.0-S0022519319304679-main.pdf)


---

[26] Measurement of Conditional Relatedness Between Genes Using Fully Convolutional Neural Network.
Yan Wang, Shuangquan Zhang, Lili Yang, Sen Yang, Yuan Tian<sup>$</sup>, **Qin Ma**<sup>$</sup>
*Frontiers in Genetics*
[[Abstract]](https://www.frontiersin.org/articles/10.3389/fgene.2019.01009/full?&utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Genetics&id=463811) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/10/fgene-10-01009.pdf)


---

[25] Single-cell RNA sequencing of plant-associated bacterial communities.
**Qin Ma**, Heike Bücking, Jose L. Gonzalez Hernandez, Sen Subramanian<sup>$</sup>
*Frontiers in Microbiology*
[[Abstract]](https://www.frontiersin.org/articles/10.3389/fmicb.2019.02452/full) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/10/fmicb-10-02452.pdf)


---

[24] LTMG: A novel statistical modeling of transcriptional expression states in single-cell RNA-Seq data.
Changlin Wan, Wennan Chang, Yu Zhang, Fenil Shah, Xiaoyu Lu, Yong Zang, Anru Zhang, Sha Cao, Melissa Fishel, **Qin Ma**<sup>$</sup>, Chi Zhang<sup>$</sup>
*Nucleic Acids Research*
[[Abstract]](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz655/5542876?rss=1) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/LTMG.pdf)

---

[23] Prediction of Regulatory Motifs from Human ChIP-Sequencing Data using a Deep Learning Framework.
Jinyu Yang, Anjun Ma, Adam D. Hoppe, Cankun Wang, Yang Li, Chi Zhang, Yan Wang, Bingqiang Liu<sup>$</sup>, **Qin Ma**<sup>$</sup>
*Nucleic Acids Research*
[[Abstract]](https://academic.oup.com/nar/article/47/15/7809/5542889?guestAccessKey=1ae813b7-d8bb-4866-b51c-343583591b94) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/DESSO.pdf)

---

[22] IRIS-EDA: An integrated RNA-Seq interpretation system for gene expression data analysis. 
Brandon Monier<sup>\*</sup>, Adam McDermaid<sup>\*</sup>, Cankun Wang, Jing Zhao, Allison Miller, Anne Fennell, **Qin Ma**<sup>$</sup>
*PLoS Computational Biology*
[[Abstract]](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006792) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-IRIS-EDA-An-integrated-RNA-Seq-interpretation-system-for-gene-expression-data-analysis.pdf)

---

[21] SubMito-XGBoost: predicting protein submitochondrial localization by fusing multiple feature information and eXtreme gradient boosting.
Bin Yu<sup>$</sup>, Wenying Qiu, Cheng Chen, Anjun Ma, Jing Jiang, Hongyan Zhou, **Qin Ma**<sup>$</sup>
*Bioinformatics*
[[Abstract]](https://academic.oup.com/bioinformatics/article/36/4/1074/5585744?guestAccessKey=73ca5b12-d4a6-4a0f-bfff-447606f0f7a3) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/10/btz734.pdf)

---

[20] QUBIC2: A novel and robust biclustering algorithm for analyses and interpretation of large-scale RNA-Seq data.
Juan Xie, Anjun Ma, Yu Zhang, Bingqiang Liu, Sha Cao, Cankun Wang, Jennifer Xu, Chi Zhang<sup>$</sup>, **Qin Ma**<sup>$</sup>
*Bioinformatics*
[[Abstract]](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz692/5567116) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/09/qubic2.pdf)


---

[19] Protein-protein interaction sites prediction by ensembling random forests with synthetic minority oversampling technique.
Xiaoying Wang<sup>\*</sup>, Bin Yu<sup>$</sup><sup>\*</sup>, Anjun Ma, Cheng Chen, Bingqiang Liu, **Qin Ma**<sup>$</sup>
*Bioinformatics*
[[Abstract]](https://academic.oup.com/bioinformatics/article/35/14/2395/5231852) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-Protein-protein-interaction-sites-prediction-by-ensembling-random-forests-with-synthetic-minority-oversampling-technique.pdf)

---

[18] MetaQUBIC: a computational pipeline for gene-level functional profiling of metagenome and metatranscriptome.
Anjun Ma<sup>\*</sup>, Minxuan Sun<sup>\*</sup>, Adam McDermaid, Bingqiang Liu<sup>$</sup>, **Qin Ma**<sup>$</sup>
*Bioinformatics*
[[Abstract]](https://academic.oup.com/bioinformatics/article/35/21/4474/5497255) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/MetaQUBIC.pdf)

---

[17] MetaMed: Linking Microbiota Functions with Medicine Therapeutics.
Han Zhao, Shaliu Fu, Yifei Yu, Zhanbing Zhang, Ping Li, **Qin Ma**, Wei Jia, Kang Ning, Shen Qu, Qi Liu<sup>$</sup>
*American Society for Microbiology*
[[Abstract]](https://msystems.asm.org/content/4/5/e00413-19) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/10/mSystems-2019-Zhao-e00413-19.full_.pdf)


---

[16] Predicting Golgi-resident Protein Types Using Conditional Covariance Minimization with XGBoost Based on Multiple Features Fusion.
Hongyan Zhou, Cheng Chen, Minghui Wang, **Qin Ma**, Bin Yu<sup>$</sup>
*IEEE Access*
[[Abstract]](https://ieeexplore.ieee.org/document/8817940) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/08817940.pdf)


---

[15] Identification of lncRNAs-gene interactions in transcription regulation based on co-expression analysis of RNA-seq data.
Sijie Lu, Juan Xie, Yang Li, Bin Yu, **Qin Ma**<sup>$</sup>, Bingqiang Liu<sup>$</sup>
*Mathematical Biosciences and Engineering*
[[Abstract]](https://www.aimspress.com/article/10.3934/mbe.2019357) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/Identification-of-lncRNAs-gene-interactions.pdf)


---

[14] Computational Prediction of Sigma-54 Promoters in Bacterial Genomes by Integrating Motif Finding and Machine Learning Strategies.
Bingqiang Liu , Ling Han, Xiangrong Liu , Jichang Wu, **Qin Ma**<sup>$</sup>
*EEE/ACM Transactions on Computational Biology and Bioinformatics*
[[Abstract]](https://ieeexplore.ieee.org/document/8316939) [[Full paper]](https://u.osu.edu/bmbl/files/2020/08/08316939.pdf)


---

[13] Clustering and Classification Methods for Single-cell RNA-sequencing Data.
Ren Qi, Anjun Ma, **Qin Ma**, Quan Zou<sup>$</sup>
*Briefings in Bioinformatics*
[[Abstract]](https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbz062/5528236) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/201908BIB-Clustering-and-classification-methods-for-single-cell-RNA-Seq-data.pdf)


---

[12] LightGBM-PPI: Predicting protein-protein interactions through LightGBM with multi-information fusion
Cheng Chen, Qingmei Zhang, **Qin Ma**, Bin Yu<sup>$</sup>
*Chemometrics and Intelligent Laboratory Systems*
[[Abstract]](https://www.sciencedirect.com/science/article/pii/S016974391930262X) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/10/1-s2.0-S016974391930262X-main.pdf)


---

[11] Transcriptomic response is more sensitive to water deficit in shoots than roots of Vitis riparia (Michx.).
Vedbar Singh Khadka, Kimberley Vaughn, Juan Xie, Padmapriya Swaminathan, **Qin Ma**, Grant R. Cramer, Anne Y. Fennell<sup>$</sup>
*BMC Plant Biology*
[[Abstract]](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-019-1664-7) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/01/2018-Transcriptomic-response-is-more-sensitive-to-water-deficit-in-shoots-than-roots-of-Vitis-riparia-Michx.-13zj21t.pdf)


---

[10] Differential Gene Expression in Non-Transgenic and Transgenic 'M.26' Apple Overexpressing a Peach CBF Gene During the Transition from Eco-Dormancy to Bud Break.
Timothy Artlip, Adam McDermaid, **Qin Ma**, Michael E Wisniewski<sup>$</sup>
*Horticulture Research*
[[Abstract]](https://www.nature.com/articles/s41438-019-0168-9) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/10/s41438-019-0168-9.pdf)


---

[9] Using Machine Learning to Measure Relatedness Between Genes: A Multi-Features Model.
Yan Wang, Sen Yang, Jing Zhao, Wei Du, Yanchun Liang, Cankun Wang, Fengfeng Zhou, Yuan Tian, **Qin Ma**<sup>$</sup>
*Scientific Reports*
[[Abstract]](https://www.nature.com/articles/s41598-019-40780-7) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-Using-Machine-Learning-to-Measure-Relatedness-Between-Genes-A-Multi-Features-Model.pdf)


---

[8] Predicting drug-target interactions using Lasso with random forest based on evolutionary information and chemical structure.
Han Shia, Simin Liu, Junqi Chen, Xuan Li, **Qin Ma**<sup>$</sup>, Bin Yu<sup>$</sup>
*Genomics*
[[Abstract]](https://www.sciencedirect.com/science/article/pii/S088875431830466X?via%3Dihub) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-Predicting-drug-target-interactions-using-Lasso-with-random-forest-based-on-evolutionary-information-and-chemical-structure.pdf)


---

[7] GmZPR3d interacts with GmHD-ZIP III proteins and regulates soybean root and nodule vascular development.
Suresh Damodaran, Amélie Dubois, Juan Xie, **Qin Ma**, Valérie Hindié, Senthil Subramanian<sup>$</sup>
*International Journal of Molecular Sciences*
[[Abstract]](https://www.mdpi.com/1422-0067/20/4/827) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-GmZPR3d-Interacts-with-GmHD-ZIP-III-Proteins-and-Regulates-Soybean-Root-and-Nodule-Vascular-Development.pdf)


---

[6] Removable edges on a Hamilton cycle or outside a cycle in a 4-connected graph.
Jichang Wu, Hajo Broersma, Yaping Mao, Qin Ma<sup>$</sup>
*Discussiones Mathematicae Graph Theory (DMGT)*
[[Abstract]](https://www.dmgt.uz.zgora.pl/publish/view_press.php?ID=2404) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-Removable-edges-on-a-Hamilton-cycle-or-outside-a-cycle-in-a-4-connected-graph.pdf)


---

[5] Improved Draft Genome Sequence of Bacillus sp. Strain YF23, Which Has Plant Growth-Promoting Activity.
Ye Xia, Seth Debolt, **Qin Ma**, Adam McDermaid, Cankun Wang, Nicole Shapiro, Tanja Woyke, Nikos C. Kyrpides<sup>$</sup>
*Microbiology Resource Announcements*
[[Abstract]](https://mra.asm.org/content/8/15/e00099-19.abstract) [[Full paper]](https://mra.asm.org/content/ga/8/15/e00099-19.full.pdf)


---

[4] Improved Draft Genome Sequence of Pseudomonas poae strain A2-S9 with plant growth promotion activity.
Ye Xia, Seth Debolt, **Qin Ma**, Adam McDermaid, Cankun Wang, Nicole Shapiro, Tanja Woyke, Nikos C. Kyrpides<sup>$</sup>
*Microbiology Resource Announcements*
[[Abstract]](https://mra.asm.org/content/8/15/e00275-19) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-Improved-Draft-Genome-Sequence-of-Pseudomonas-poae-strain-A2-S9-with-plant-growth-promotion-activity.pdf)


---

[3] rSeqTU – a machine-learning based R package for prediction of bacterial transcription units.
YSheng-Yong Niu, Bingqiang Liu, **Qin Ma**, Wen-Chi Chou<sup>$</sup>
*Frontiers in Genetics*
[[Abstract]](https://www.frontiersin.org/articles/10.3389/fgene.2019.00374/full) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-rSeqTU%E2%80%94A-Machine-Learning-Based-R-Package-for-Prediction-of-Bacterial-Transcription-Units.pdf)


---

[2] Rootstock effects on scion phenotypes in a ‘Chambourcin’ experimental vineyard.
Allison Miller, Zoë Migicovsky, Zachary Harris, Laura Klein, Mao Li, Adam McDermaid, Daniel Chitwood, Anne Fennell, Laszlo Kovacs, Misha Kwasniewski, Jason Londo, and **Qin Ma**<sup>$</sup>
*Horticulture Research*
[[Abstract]](https://www.nature.com/articles/s41438-019-0146-2) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-Rootstock-effects-on-scion-phenotypes-in-a-Chambourcin-experimental-vineyard.pdf)


---

[1] Pancreatic Cancer Biomarker Detection by Two Support Vector Strategies for Recursive Feature Elimination.
Yan Wang, Keke Liu, **Qin Ma**, Yongfei Tan, Wei Du, Yidan Lv, Yuan Tia<sup>$</sup>, Hao Wang<sup>$</sup>
*Biomarkers in Medicine*
[[Abstract]](https://www.futuremedicine.com/doi/10.2217/bmm-2018-0273) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/01/2019sci-wy-Pancreatic-Cancer-Biomarker-Detection-by-Two-Support-Vector-Strategies-for-Recursive-Feature-Elimination-2eavkdq.pdf)

