<h3>2021</h3>

<hr />

[04/26/2021] Yang Li, <a href="https://u.osu.edu/bmbl/files/2021/04/Expression-accessibility-heterogeneity-kidney-1.pptx">Single cell transcriptional and chromatin accessibility profiling redefine cellular heterogeneity in the adult human kidney</a>

[04/19/2021] Anjun Ma, <a href="https://u.osu.edu/bmbl/files/2021/04/CellChat-paper-review.pptx">Inference and analysis of cell-cell communication using CellChat</a>

[03/10/2021] Xiaoying Wang, <a href="https://u.osu.edu/bmbl/files/2021/03/totalVi-Xiaoying.pptx">totalVI: Joint probabilistic modeling of single-cell multi-omic data</a>

[03/10/2021] Yujia Xiang, <a href="https://u.osu.edu/bmbl/files/2021/03/LiteratureReview-Stein-Aerts-Yujia-20210222Latest.pptx">Stein Aerts lab</a>

[03/08/2021] Yuzhou Chang&Cankun Wang, <a href="https://u.osu.edu/bmbl/files/2021/03/Computation-Resources_2021_update.pptx">BMBL Computing Resources 2021</a>

[02/15/2021] Wantong Li, <a href="https://u.osu.edu/bmbl/files/2021/02/2021.02.15Cell-Paper-Perturb-ATAC.pptx">Coupled Single-Cell CRISPR Screening and Epigenomic Profiling Reveals Causal Gene Regulatory Networks</a>

[01/15/2021] Zhenyu Wu, <a href="https://u.osu.edu/bmbl/files/2021/01/2021.01.24LISA-Liu-Shirley.pptx">Lisa: Inferring transcriptional regulators through integrative modeling of public chromatin accessibility and ChIP-seq data</a>

[01/18/2021] Yuzhou Chang, <a href="https://u.osu.edu/bmbl/files/2021/01/Computation-Resources_2021_update.pptx">Computation resources 2021 update</a>

[01/11/2021] Yang Li, <a href="https://u.osu.edu/bmbl/files/2021/01/Algorithmic-Analytics-01112021.pptx">Towards algorithmic analytics for large-scale datasets </a>

<h3>2020</h3>

<hr />
[12/10/2020] Anjun Ma, <a href="https://u.osu.edu/bmbl/files/2020/12/NC-Machine-learning-uncovers-cell-identity-regulator-by-histone-code-12082020.pptx">MARS: discovering novel cell types acrossheterogeneous single-cell experiments</a>

[12/02/2020] Jing Jiang, <a href="https://u.osu.edu/bmbl/files/2020/12/Single-cell-RNA-sequencing-12022020.pptx">Single cell RNA sequencing of human microglia uncovers a subset associated with Alzheimer’s disease</a>

[12/02/2020] Zhaoqian Liu, <a href="https://u.osu.edu/bmbl/files/2020/12/Lab-Meeting-ZQ_12022020.pptx">A unified catalog of 204,938 reference genomes from the human gut microbiome</a>

[11/23/2020] Yuzhou Chang, <a href="https://u.osu.edu/bmbl/files/2020/12/Yuzhou-presentation-omics-Autosaved.pptx">High-Spatial-Resolution Multi-Omics Sequencing via Deterministic Barcoding in Tissue</a>

[10/28/2020] Zhenyu Wu, <a href="https://u.osu.edu/bmbl/files/2020/12/2020.10.28cell-type-specific-epigenome.pptx">Cell-type-specific 3D epigenomes in the developing human cortex
</a>

[10/14/2020] Ren Qi, <a href="https://u.osu.edu/bmbl/files/2020/10/Lab-Meeting-Ren_10142020.pptx">Ensemble deep learning in bioinformatics</a>

[10/14/2020] Zhaoqian Liu, <a href="https://u.osu.edu/bmbl/files/2020/10/Lab-Meeting-ZQ_10142020-v2.pptx">Gut microbiota in human metabolic health and disease</a>

[10/14/2020] Jing Jiang, <a href="https://u.osu.edu/bmbl/files/2020/10/Single-cell-transcriptomic-analysis-of-AD-10142020.pptx">Single-cell transcriptomic analysis- of AD</a>

[09/23/2020] Yuzhou Chang, <a href="https://u.osu.edu/bmbl/files/2020/09/Yuzhou-presentation-omics.pptx">snRNA-Seq, spatial transcriptomics, and deep learning</a>

[09/23/2020] Cankun Wang, <a href="https://u.osu.edu/bmbl/files/2020/09/Cankun_Lab_meeting_09232020.pptx">Identification of TFBS in ATAC-seq</a>

[09/16/2020] Mirian Alvarez, <a href="https://u.osu.edu/bmbl/files/2020/09/scAlign.pptx">Introduction to scAlign</a>

[09/02/2020] Yang Li, <a href="https://u.osu.edu/bmbl/files/2020/09/2020_Nature_ENCODE3_09012020_Yang.pptx">ENCODE 3</a>

[09/02/2020] Yuzhou Chang, <a href="https://u.osu.edu/bmbl/files/2020/09/Yuzhou-presentation-spatial.pptx">snRNA-Seq, spatial transcriptomics, and deep learning</a>

[09/02/2020] Cankun Wang, <a href="https://u.osu.edu/bmbl/files/2020/09/Cankun_Lab_meeting_t_cell_09022020.pptx">Global characterization of T cells in non-small-cell lung cancer by single-cell sequencing</a>

[08/24/2020] Jing Jiang, <a href="https://u.osu.edu/bmbl/files/2020/08/Systematic-comparison-of-single-cell-and-single-nucleus-RNA-sequencing-methods-Jing-1.pptx">Systematic comparison of single-cell nad single-nucleus RNA-sequencing methods</a>

[08/03/2020] Yuzhou Chang, <a href="https://u.osu.edu/bmbl/files/2020/08/Computation-Resources_2020_update.pptx">Computation Resource Updates</a>

[07/27/2020] Yuzhou Chang, <a href="https://u.osu.edu/bmbl/files/2020/07/Spatial_scRNA-Seq.pptx">Spatial Transcriptomics Project</a>

[07/20/2020] Cankun Wang, <a href="https://u.osu.edu/bmbl/files/2020/08/Cankun_Lab_meeting_CSOmap_07202020.pptx">Reconstruction of Cell Spatial Organization From Single-cell RNA Sequencing Data Based on Ligand-receptor Mediated Self-assembly
</a>

[07/13/2020] Zhenyu Wu, <a href="https://u.osu.edu/bmbl/files/2020/07/2020.07.13ChromID.pptx">ChromID Identifies the Protein Interactome at Chromatin Marks</a>

[07/13/2020] Patrick Stevens, <a href="https://u.osu.edu/bmbl/files/2020/06/Lab-Meeting-ZQ-0629.pptx">Assessment of Computational Methods for the Analysis of Single-cell ATAC-seq Data</a>

[06/29/2020] Zhaoqian Liu, <a href="https://u.osu.edu/bmbl/files/2020/06/Lab-Meeting-ZQ-0629.pptx">Complete, Closed Bacterial Genomes From Microbiomes Using Nanopore Sequencing
</a>

[06/29/2020] Yuzhou Chang, <a href="https://u.osu.edu/bmbl/files/2020/06/BRIC.pptx">BRIC Project: Biclustering-based Gene Regulation Inference and Cell Type Prediction</a>

[06/29/2020] Patrick Stevens, <a href="https://u.osu.edu/bmbl/files/2020/06/scATAC_Cistopic_analysis.pptx">Cistopic Analysis on scATAC Thymocyte data</a>

[06/18/2020] Yang Li, <a href="https://u.osu.edu/bmbl/files/2020/12/AI-Applications-2019-NRG-Yang-06182020.pptx">Deep learning: new computational modelling techniques for genomics</a>

[05/29/2020] 10X genomic seminar, <a href="https://www.youtube.com/watch?v=W5AN534PMtM">https://www.youtube.com/watch?v=W5AN534PMtM</a>

[05/27/2020] Zhenyu Wu,<a href="https://u.osu.edu/bmbl/files/2020/05/HTAN.pptx"> The Human Tumor Atlas Network: Charting Tumor Transitions across Space and Time at Single-Cell Resolution</a>

[05/27/2020] Jing Jiang, <a href="https://u.osu.edu/bmbl/files/2020/05/HTAN.pptx">The Human Tumor Atlas Network: Charting Tumor Transitions across Space and Time at Single-Cell Resolution</a>

[04/29/2020] Ren Qi, <a href="https://u.osu.edu/bmbl/files/2020/04/GNN-RenQi-04292020.pptx">GNN intro</a>

[04/29/2020] Zhaoqian Liu, <a href="https://u.osu.edu/bmbl/files/2020/04/Lab-Meeting-ZQ-04292020-GNN.pptx">GNN Examples</a>

[04/27/2020] Cankun Wang, <a href="https://u.osu.edu/bmbl/files/2020/04/RNA-seq_analysis_Cankun_04262020.pptx">RNA-seq Analysis</a>

[04/27/2020] Yang Li, PhD, <a href="https://u.osu.edu/bmbl/files/2020/04/QUBIC-GO-For-Presentation-04272020.pptx">QUBIC-GO-For-Presentation</a>

[04/14/2020] Yang Li, PhD, <a href="https://u.osu.edu/bmbl/files/2020/04/GAN-Application-scRNA-Seq-YangLi-03302020.pptx">GAN-Application-scRNA-Seq</a>

[03/23/2020] Jing Zhao, PhD, <a href="https://u.osu.edu/bmbl/files/2020/03/CNN-RNN-examples-in-EMR.pptx">CNN RNN examples in EMR</a>

[03/23/2020] Zhaoqian Liu, <a href="https://u.osu.edu/bmbl/files/2020/03/RNN-ZQ-03232020.pptx">RNN Introduction</a>

[03/23/2020] Jing Jiang, <a href="https://u.osu.edu/bmbl/files/2020/03/Recurrent-Neural-Network-Applications.pptx">Recurrent Neural Network Applications</a>

[03/16/2020] Ren Qi, <a href="https://u.osu.edu/bmbl/files/2020/03/CNN-RenQi-03162020-V3.pptx">Introduction to CNN</a>

[03/16/2020] Cankun Wang, <a href="https://u.osu.edu/bmbl/files/2020/03/Cankun_Lab_meeting_CNN_example_03162020.pptx">Implementation of CNN using DESSO</a>

[03/09/2020] Yang Li, <a href="https://u.osu.edu/bmbl/files/2020/03/DNN-YangLi-03092020-V3.pptx">Deep Neural Network</a>

[03/09/2020] Yuzhou Chang, <a href="https://u.osu.edu/bmbl/files/2020/03/GNN_applicationLAB_meeting_Mar_09_2020.pptx">Deep Neuron Network Application - 6 Papers Apply DNN on Single cell Level Analysis</a>

[03/03/2020] Zhenyu Wu, <a href="https://u.osu.edu/bmbl/files/2020/03/2020.03.03lab-pre-of-zhenyu-Wu.pptx">Quantified Pathway Analysis</a>

[03/03/2020] Cankun Wang, <a href="https://u.osu.edu/bmbl/files/2020/03/Cankun_Lab_meeting_03032020.pptx">Introduction to Deep Learning Tools</a>

[03/03/2020] Qin Ma and Jing Zhao, <a href="https://u.osu.edu/bmbl/files/2020/03/Introduction-to-AI-ML-and-DL03032020.pptx">Introduction to Deep Learning</a>

[03/03/2020] Zhaoqian Liu, <a href="https://u.osu.edu/bmbl/files/2020/03/Lab-Meeting-ZQ-0303.pptx">Relating the gut metagenome and metatranscriptome to immunotherapy responses in melanoma patients</a>

[02/17/2020] Cankun Wang, <a href="https://u.osu.edu/bmbl/files/2020/02/Cankun_Lab_meeting_02172020.pptx">Cicero Predicts cis-Regulatory DNA Interactions From Single-Cell Chromatin Accessibility Data</a>

[02/10/2020] Yang Li, PhD, <a href="https://u.osu.edu/bmbl/files/2020/02/Presentation-YangLi-01102020.pptx">Sub-modular optimization</a>

[01/27/2020] Jing Jiang, <a href="https://u.osu.edu/bmbl/files/2020/02/ENHANCER-GENOMICS.pptx">Brain cell type–specific enhancer–promoter interactome maps and disease-risk association.</a>

[01/27/2020] Zhaoqian Liu, <a href="https://u.osu.edu/bmbl/files/2020/02/Lab-Meeting-ZQ-0120.pptx">Learning representations of microbe-metabolite interactions.</a>

[01/13/2020] Yuzhou Chang, <a href="https://u.osu.edu/bmbl/files/2020/02/LabMeeting1_13_2020.pptx">Seurat: Overview of three major versions.</a>

[01/13/2020] Cankun Wang, <a href="https://u.osu.edu/bmbl/files/2020/02/Cankun_Lab_meeting_01132019.pptx">A massively parallel reporter assay dissects the influence of chromatin structure on cis-regulatory activity.</a>

[01/06/2020] Yang Li, <a href="https://u.osu.edu/bmbl/files/2020/02/SCENIC-YLi-01062020.pptx">SCENIC: single-cell regulatory network inference and clustering</a>.

[01/06/2020] Anjun Ma, <a href="https://u.osu.edu/bmbl/files/2020/02/Anjun-01062020.pptx">From Single-cell RNA-Seq to transcriptional regulation.</a>

<h3>2018</h3>

<hr />

[04/17/2018] Cankun Wang, The introduction of Circos.

[01/29/2018] Juan Xie, <a href="https://u.osu.edu/bmbl/files/2019/01/01-2018-juan-visualize-gene-expression-in-Cytoscape-1xntkzv.pdf">Cytoscape: Visualize gene expression in Cytoscape</a>.

[01/16/2018] Shihan Wu, <a href="https://u.osu.edu/bmbl/files/2019/01/shihan-wu-ML10108-1gthogf.pdf">Machine learning extracts patterns from data without explicit instructions</a>.


<h3>2017</h3>

<hr />

[09/01/2017] Juan Xie, <a href="https://u.osu.edu/bmbl/files/2019/01/09-01-17-Juan-Xsede-2017-1x7cekv.pdf">Introduction to XSEDE/bridges</a>.

[08/28/2017] Anjun Ma, <a href="https://u.osu.edu/bmbl/files/2019/01/Anjun-Ma-Journal-Club-082817-rz043s.pdf">Intro: Investigation of Candidate Loci Associated with Maize Perennialism</a>.

[08/21/2017] Juan Xie, <a href="https://u.osu.edu/bmbl/files/2019/01/LCS1-29o2m03.pdf">Longest Common Subsequence (LCS)</a>.

[08/04/2017] Alexander Harsh, <a href="https://u.osu.edu/bmbl/files/2019/01/Sea-Wheat-Grass-DNA-assembly-on-bridges-using-meraculous-18dz94q.pdf">Sea Wheat Grass DNA Assembly on Bridges Using </a>Meraculous.

[06/21/2017] Alexander Harsh, <a href="https://u.osu.edu/bmbl/files/2019/01/Meraculous-18dp13t.pdf">Meraculous</a>.

[05/08/2017] Adam McDermaid, <a href="https://u.osu.edu/bmbl/files/2019/01/De-Bruijn-Graphs-and-Applications-to-Assembly-13tc7fg.pdf">De Bruijn Graphs and Applications to Assembly</a>.

[05/08/2017] Jinyu Yang, Regulatory DNA motif identification by integrating DNA shape in a deep learning framework.

[04/24/2017] Adam McDermaid, <a href="https://u.osu.edu/bmbl/files/2019/01/Toil-open-source-workflow-1es87f2.pdf">TOIL open source workflow software</a>.

[04/03/2017] Adam McDermaid and Juan Xie, <a href="https://u.osu.edu/bmbl/files/2019/01/p_value-04032017-19nelqu.pdf">Significance, P-value and t-tests</a>.

<hr />

<h3>2016</h3>

<hr />

[08/12/2016] Adam McDermaid, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-08-12-GeneQC-Statistical-Model-Presentation-modified-1nk2ggk.pdf">GeneQC Statistical Model</a>.

[08/05/2016] Juan Xie, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-08-05-Quantitative-modeling-of-gene-expression-using-DNA-shape-features-of-binding-sites-2mk5age.pdf">Quantitative modeling of gene expression using DNA shape features of binding sites</a>.

[07/25/2016] Jinyu Yang, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-07-25-Convolutional-Neural-Network-for-DNA-protein-Prediction-xjd47f.pdf">Convolutional Neural Network for DNA-protein Prediction</a>.

[07/15/2016] Jinyu Yang, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-07-15-Metagenomics-and-Metatranscriptomics-21nb0ug.pdf">Metagenomics and Metatranscriptomics</a>.

[07/08/2016] Adam McDermaid, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-07-08-Differential-Gene-Expression-Analysis-Using-DESeq-29q3u7d.pdf">Differential Gene Expression Analysis Using DESeq</a>.

[07/01/2016] Adam McDermaid, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-07-01-Principal-component-analysis-network-component-analysis-2mew3vh.pdf">Principal component analysis &amp; network component analysis</a>.

[06/24/2016] Juan Xie, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-06-24-Introduction-to-Greenfield-1rx7v7q.pdf">Introduction to Greenfield</a>.

[06/10/2016] Jinyu Yang, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-06-10-Introduction-to-iPlant-288kacg.pdf">iPlant collaborative</a>.

[06/06/2016] Yiran Zhang, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-06-06-How-to-do-differential-expression-analysis-from-Fastq-24nptxz.pdf">How to do differential expression analysis from Fastq data</a>.

[05/15/2016] Jinyu Yang, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-05-15-Deep-Learning-2cajb0h.pdf">Deep Learning</a>.

[05/13/2016] Juan Xie, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-05-13-Introduction-to-XSEDE-resources-1uh82ti.pdf">Introduction to XSEDE resources</a>.

[04/29/2016] Qin Ma, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-04-29-How-to-carry-out-bioinformatic-studies-from-computational-point-of-view-23wcqri.pdf">How to carry out bioinformatic studies from a computational point of view</a>.

[04/22/2016] Yiran Zhang, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-04-22-how-to-use-RNA-sequences-analysis-tools-on-HPC-wejzz3.pdf">Using RNA Sequence Tools</a>.

[04/08/2016] Adam McDermaid, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-04-08-HISAT-a-fast-spliced-aligner-with-low-memory-requirements-ynuirz.pdf">HISAT: a fast spliced aligner with low memory requirements</a>.

[03/18/2016] Jinyu Yang, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-03-18-Predicting-the-sequence-specificities-of-DNA-and-RNA-binding-proteins-by-deep-learning-1xugui1.pdf">Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning</a>.

[03/11/2016] Juan Xie, <a href="https://u.osu.edu/bmbl/files/2019/01/2016-03-11-An-ensemble-biclustering-approach-for-querying-gene-expression-compendia-with-experimental-lists-2djzgbm.pdf">An ensemble biclustering approach for querying gene expression compendia with experimental lists</a>.
<div class="ms-editor-squiggler" style="color: initial; font: initial; font-feature-settings: initial; font-kerning: initial; font-optical-sizing: initial; font-variation-settings: initial; forced-color-adjust: initial; text-orientation: initial; text-rendering: initial; -webkit-font-smoothing: initial; -webkit-locale: initial; -webkit-text-orientation: initial; -webkit-writing-mode: initial; writing-mode: initial; zoom: initial; place-content: initial; place-items: initial; place-self: initial; alignment-baseline: initial; animation: initial; appearance: initial; aspect-ratio: initial; backdrop-filter: initial; backface-visibility: initial; background: initial; background-blend-mode: initial; baseline-shift: initial; block-size: initial; border-block: initial; border: initial; border-radius: initial; border-collapse: initial; border-end-end-radius: initial; border-end-start-radius: initial; border-inline: initial; border-start-end-radius: initial; border-start-start-radius: initial; inset: initial; box-shadow: initial; box-sizing: initial; break-after: initial; break-before: initial; break-inside: initial; buffered-rendering: initial; caption-side: initial; caret-color: initial; clear: initial; clip: initial; clip-path: initial; clip-rule: initial; color-interpolation: initial; color-interpolation-filters: initial; color-rendering: initial; color-scheme: initial; columns: initial; column-fill: initial; gap: initial; column-rule: initial; column-span: initial; contain: initial; contain-intrinsic-size: initial; content: initial; content-visibility: initial; counter-increment: initial; counter-reset: initial; counter-set: initial; cursor: initial; cx: initial; cy: initial; d: initial; display: block; dominant-baseline: initial; empty-cells: initial; fill: initial; fill-opacity: initial; fill-rule: initial; filter: initial; flex: initial; flex-flow: initial; float: initial; flood-color: initial; flood-opacity: initial; grid: initial; grid-area: initial; height: 0px; hyphens: initial; image-orientation: initial; image-rendering: initial; inline-size: initial; inset-block: initial; inset-inline: initial; isolation: initial; letter-spacing: initial; lighting-color: initial; line-break: initial; list-style: initial; margin-block: initial; margin: initial; margin-inline: initial; marker: initial; mask: initial; mask-type: initial; max-block-size: initial; max-height: initial; max-inline-size: initial; max-width: initial; min-block-size: initial; min-height: initial; min-inline-size: initial; min-width: initial; mix-blend-mode: initial; object-fit: initial; object-position: initial; offset: initial; opacity: initial; order: initial; origin-trial-test-property: initial; orphans: initial; outline: initial; outline-offset: initial; overflow-anchor: initial; overflow-wrap: initial; overflow: initial; overscroll-behavior-block: initial; overscroll-behavior-inline: initial; overscroll-behavior: initial; padding-block: initial; padding: initial; padding-inline: initial; page: initial; page-orientation: initial; paint-order: initial; perspective: initial; perspective-origin: initial; pointer-events: initial; position: initial; quotes: initial; r: initial; resize: initial; ruby-position: initial; rx: initial; ry: initial; scroll-behavior: initial; scroll-margin-block: initial; scroll-margin: initial; scroll-margin-inline: initial; scroll-padding-block: initial; scroll-padding: initial; scroll-padding-inline: initial; scroll-snap-align: initial; scroll-snap-stop: initial; scroll-snap-type: initial; shape-image-threshold: initial; shape-margin: initial; shape-outside: initial; shape-rendering: initial; size: initial; speak: initial; stop-color: initial; stop-opacity: initial; stroke: initial; stroke-dasharray: initial; stroke-dashoffset: initial; stroke-linecap: initial; stroke-linejoin: initial; stroke-miterlimit: initial; stroke-opacity: initial; stroke-width: initial; tab-size: initial; table-layout: initial; text-align: initial; text-align-last: initial; text-anchor: initial; text-combine-upright: initial; text-decoration: initial; text-decoration-skip-ink: initial; text-indent: initial; text-overflow: initial; text-shadow: initial; text-size-adjust: initial; text-transform: initial; text-underline-offset: initial; text-underline-position: initial; touch-action: initial; transform: initial; transform-box: initial; transform-origin: initial; transform-style: initial; transition: initial; user-select: initial; vector-effect: initial; vertical-align: initial; visibility: initial; -webkit-app-region: initial; border-spacing: initial; -webkit-border-image: initial; -webkit-box-align: initial; -webkit-box-decoration-break: initial; -webkit-box-direction: initial; -webkit-box-flex: initial; -webkit-box-ordinal-group: initial; -webkit-box-orient: initial; -webkit-box-pack: initial; -webkit-box-reflect: initial; -webkit-highlight: initial; -webkit-hyphenate-character: initial; -webkit-line-break: initial; -webkit-line-clamp: initial; -webkit-mask-box-image: initial; -webkit-mask: initial; -webkit-mask-composite: initial; -webkit-perspective-origin-x: initial; -webkit-perspective-origin-y: initial; -webkit-print-color-adjust: initial; -webkit-rtl-ordering: initial; -webkit-ruby-position: initial; -webkit-tap-highlight-color: initial; -webkit-text-combine: initial; -webkit-text-decorations-in-effect: initial; -webkit-text-emphasis: initial; -webkit-text-emphasis-position: initial; -webkit-text-fill-color: initial; -webkit-text-security: initial; -webkit-text-stroke: initial; -webkit-transform-origin-x: initial; -webkit-transform-origin-y: initial; -webkit-transform-origin-z: initial; -webkit-user-drag: initial; -webkit-user-modify: initial; white-space: initial; widows: initial; width: initial; will-change: initial; word-break: initial; word-spacing: initial; x: initial; y: initial; z-index: initial;"></div>