\* These authors contributed equally to the paper as first authors
$ To whom correspondence should be addressed 


---

[24] Analysis of next- and third-generation RNA-Seq data reveals the structures of alternative transcription units in bacterial genomes.
Qi Wang, Zhaoqian Liu, Bo Yan, Wen-Chi Chou, Laurence M Ettwiller, **Qin Ma**, Bingqiang Liu<sup>$</sup>
*bioRxiv*
[[Abstract]](https://www.biorxiv.org/content/10.1101/2021.01.02.425006v1) [[Full paper]](https://www.biorxiv.org/content/10.1101/2021.01.02.425006v1.full.pdf)


---

[23] Assessing deep learning algorithms in cis-regulatory motif finding based on genomic sequencing data.
Yan Wang, Shuangquan Zhang, Anjun Ma, Cankun Wang, Zhenyu Wu, Dong Xu, **Qin Ma**<sup>$</sup>
*bioRxiv* (2020)
[[Abstract]](https://www.biorxiv.org/content/10.1101/2020.11.30.403261v1) [[Full paper]](https://www.biorxiv.org/content/10.1101/2020.11.30.403261v1.full.pdf)


---

[22] Single-cell Techniques and Deep Learning in Predicting Drug Response.
Zhenyu Wu, Patrick Lawrence, Anjun Ma, Jian Zhu, Dong Xu, **Qin Ma**<sup>$</sup>
*Trends in Pharmacological Sciences* (2020)
[[Abstract]](https://doi.org/10.1016/j.tips.2020.10.004) [[Full paper]](https://u.osu.edu/bmbl/files/2020/11/1-s2.0-S0165614720302376-main.pdf)

---

[21] Elucidation of Biological Networks Across Complex Diseases Using Single-Cell Omics.
Yang Li, Anjun Ma, Ewy A. Mathe, Bingqiang Liu, Lang Li, **Qin Ma**<sup>$</sup>
*Trends in Genetics* (2020)
[[Abstract]](https://www.sciencedirect.com/science/article/pii/S0168952520302043) [[Full paper]](https://u.osu.edu/bmbl/files/2020/08/1-s2.0-S0168952520302043-main.pdf)

---

[20] scGNN: a novel graph neural network framework for single-cell RNA-Seq analyses.  
Juexin Wang<sup>\*</sup>, Anjun Ma<sup>\*</sup>, Yuzhou Chang, Jianting Gong, Yuexu Jiang, Hongjun Fu, Cankun Wang, Ren Qi, **Qin Ma**<sup>$</sup>, Dong Xu<sup>$</sup>
*bioRxiv* (2020)
[[Abstract]](https://www.biorxiv.org/content/10.1101/2020.08.02.233569v1) [[Full paper]](https://www.biorxiv.org/content/10.1101/2020.08.02.233569v1.full.pdf)


---

[19] IRIS3: Integrated Cell-type-specific Regulon Inference Server from Single-cell RNA-Seq.
Anjun Ma<sup>\*</sup>, Cankun Wang<sup>\*</sup>, Yuzhou Chang, Faith H. Brennan, Adam McDermaid, Bingqiang Liu, Chi Zhang, Phillip G. Popovich, **Qin Ma**<sup>$</sup>
*Nucleic Acids Research* (2020)
[[Abstract]](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkaa394/5838867) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2020/05/gkaa394.pdf)


---

[18] Integrative Methods and Practical Challenges for Single-cell Multi-omics.
Anjun Ma, Adam McDermaid, Jennifer Xu, Yuzhou Chang, **Qin Ma**<sup>$</sup>
*Trends in Biotechnology*  (2020)
[[Abstract]](https://www.sciencedirect.com/science/article/pii/S0167779920300573) [[Full paper]](https://u.osu.edu/bmbl/files/2020/08/Integrative_Methods_and_Practical_Challenges.pdf)

---

[17] Prediction of Regulatory Motifs from Human ChIP-Sequencing Data using a Deep Learning Framework.
Jinyu Yang, Anjun Ma, Adam D. Hoppe, Cankun Wang, Yang Li, Chi Zhang, Yan Wang, Bingqiang Liu<sup>$</sup>, **Qin Ma**<sup>$</sup>
*Nucleic Acids Research* (2019)
[[Abstract]](https://academic.oup.com/nar/article/47/15/7809/5542889?guestAccessKey=1ae813b7-d8bb-4866-b51c-343583591b94) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/DESSO.pdf)

---

[16] LTMG: A novel statistical modeling of transcriptional expression states in single-cell RNA-Seq data.
Changlin Wan, Wennan Chang, Yu Zhang, Fenil Shah, Xiaoyu Lu, Yong Zang, Anru Zhang, Sha Cao, Melissa Fishel, **Qin Ma**<sup>$</sup>, Chi Zhang<sup>$</sup>
*Nucleic Acids Research* (2019)
[[Abstract]](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz655/5542876?rss=1) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/LTMG.pdf)

---

[15] QUBIC2: A novel and robust biclustering algorithm for analyses and interpretation of large-scale RNA-Seq data.
Juan Xie, Anjun Ma, Yu Zhang, Bingqiang Liu, Sha Cao, Cankun Wang, Jennifer Xu, Chi Zhang<sup>$</sup>, **Qin Ma**<sup>$</sup>
*Bioinformatics* (2019)
[[Abstract]](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz692/5567116) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/09/qubic2.pdf)

---

[14] IRIS-EDA: An integrated RNA-Seq interpretation system for gene expression data analysis. 
Brandon Monier<sup>\*</sup>, Adam McDermaid<sup>\*</sup>, Cankun Wang, Jing Zhao, Allison Miller, Anne Fennell, **Qin Ma**<sup>$</sup>
*PLoS Computational Biology* (2019)
[[Abstract]](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006792) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2019-IRIS-EDA-An-integrated-RNA-Seq-interpretation-system-for-gene-expression-data-analysis.pdf)


---

[13] MetaQUBIC: a computational pipeline for gene-level functional profiling of metagenome and metatranscriptome.
Anjun Ma<sup>\*</sup>, Minxuan Sun<sup>\*</sup>, Adam McDermaid, Bingqiang Liu<sup>$</sup>, **Qin Ma**<sup>$</sup>
*Bioinformatics* (2019)
[[Abstract]](https://academic.oup.com/bioinformatics/article/35/21/4474/5497255) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/MetaQUBIC.pdf)

---

[12] Clustering and Classification Methods for Single-cell RNA-sequencing Data.
Ren Qi, Anjun Ma, **Qin Ma**, Quan Zou<sup>$</sup>
*Briefings in Bioinformatics* (2019)
[[Abstract]](https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbz062/5528236) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/201908BIB-Clustering-and-classification-methods-for-single-cell-RNA-Seq-data.pdf)

---

[11] QUBIC: a Bioconductor package for qualitative biclustering analysis of gene co-expression data.
Yu Zhang<sup>\*</sup>, Juan Xie<sup>\*</sup>, Jinyu Yang, Anne Fennell, **Qin Ma**<sup>$</sup>
*Bioinformatics* (2018)
[[Abstract]](https://academic.oup.com/bioinformatics/article/33/3/450/2572040) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-QUBIC-a-bioconductor-package-for-qualitative-biclustering-analysis-of-gene-co-expression-data.pdf)

---

[10] Interpretation of differential gene expression results of RNA-Seq data: review and integration.
Adam McDermaid<sup>\*</sup>, Brandon Monier<sup>\*</sup>, Jing Zhao, Bingqiang Liu, **Qin Ma**<sup>$</sup>
*Briefings in Bioinformatics* (2018)
[[Abstract]](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bby067/5066173?guestAccessKey=41fde1ae-81d4-42c5-89fd-9db5a58f8547) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-Interpretation-of-differential-gene-expression-results-of-RNA-seq-data-review-and-integration.pdf)

---

[9] DMINDA 2.0: integrated and systematic views of regulatory DNA motif identification and analyses.
Jinyu Yang, Xin Chen, Adam McDermaid, **Qin Ma**<sup>$</sup>
*Bioinformatics* (2017)
[[Abstract]](https://academic.oup.com/bioinformatics/article/33/16/2586/3611274) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2018-DMINDA-2.0-integrated-and-systematic-views-of-regulatory-DNA-motif-identification-and-analyses.pdf)


---


[8] Analysis of strand-specific RNA-seq data using machine learning reveals the structures of transcription units in Clostridium thermocellum.
Wen-Chi Chou<sup>\*</sup>, **Qin Ma**<sup>\*</sup>, Shihui Yang, Sha Cao, Dawn M. Klingeman, Steven D. Brown, Ying Xu<sup>$</sup>
*Nucleic Acids Research* (2015)
[[Abstract]](https://academic.oup.com/nar/article/43/10/e67/2409010) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/05/2015-Analysis-of-strand-specific-RNA-seq-data-using-machine-learning-reveals-the-structures-of-transcription-units-in-Clostridium-thermocellum.pdf)


---

[7] DMINDA: An integrated web server for DNA motif identification and analysis.
**Qin Ma**<sup>\*</sup>, Hanyuan Zhang<sup>\*</sup>, Xizeng Mao, Chuan Zhou, Bingqiang Liu, Xin Chen, Ying Xu<sup>$</sup>
*Nucleic Acids Research* (2014)
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/24753419/) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/DMINDA1.pdf)

---

[6] DOOR 2.0: presenting operons and their functions through dynamic and integrated views.
Xizeng Mao<sup>\*</sup>, **Qin Ma**, Chuan Zhou, Xin Chen, Hanyuan Zhang, Jincai Yang, Fenglou Mao, Wei Lai, Ying Xu<sup>$</sup>
*Nucleic Acids Research* (2014)
[[Abstract]](https://academic.oup.com/nar/article/42/D1/D654/1045168) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/DOOR2.pdf)

---

[5] A phylogenetic model for understanding the effect of gene duplication on cancer progressing. (2013)
**Qin Ma**, Jaxk Reeves, David A. Liberles, Lili Yu, Zheng Chang, Jing Zhao, Juan Cui, Ying Xu<sup>$</sup>, Liang Liu<sup>$</sup>
*Nucleic Acids Research* (2013)
[[Abstract]](https://academic.oup.com/nar/article/42/5/2870/1060811) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/phylogenetic_model.pdf)

---

[4] Computational Analyses of Transcriptomic Data Reveal the Dynamic Organization of the E. coli Chromosome under Different Conditions.
**Qin Ma**<sup>\*</sup>, Yanbin Yin, Mark A. Shell, Han Zhang, Guojun Li, Ying Xu<sup>$</sup>
*Nucleic Acids Research* (2013)
[[Abstract]](https://academic.oup.com/nar/article/41/11/5594/2411323) [[Full paper]](https://u.osu.edu/bmbl/files/2020/08/gkt261.pdf)

---


[3] Integration of Sequence-Similarity and Functional Association Information Can Overcome Intrinsic Problems in Orthology Mapping across Bacterial Genomes.
Guojun Li, **Qin Ma**, Xizeng Mao, Yanbin Yin, Xiaoran Zhu, Ying Xu<sup>$</sup>
*Nucleic Acids Research* (2011)
[[Abstract]](https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkr766) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/integration_2011_nar.pdf)

---

[2] A new framework for identifying cis-regulatory motifs in prokaryotes.
Guojun Li, Bingqiang Liu, **Qin Ma**, Ying Xu<sup>$</sup>
*Nucleic Acids Research* (2011)
[[Abstract]](https://academic.oup.com/nar/article/39/7/e42/1151705) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/bobro1.pdf)

---


[1] QUBIC: A Qualitative Biclustering Algorithm for Analyses of Gene Expression Data.
Guojun Li, **Qin Ma**, Haibao Tang, Andrew H. Paterson, Ying Xu<sup>$</sup>
*Nucleic Acids Research* (2009)
[[Abstract]](https://pubmed.ncbi.nlm.nih.gov/19509312/) [[Full paper]](https://cpb-us-w2.wpmucdn.com/u.osu.edu/dist/0/72768/files/2019/08/qubic1.pdf)

